tests/contrib/django/test_django_snapshots.py

```diff
@@ -94,9 +94,10 @@ def psycopg2_patched(transactional_db):
 def test_psycopg_query_default(client, psycopg2_patched):
     """Execute a psycopg2 query on a Django database wrapper"""
     from django.db import connections
+    from psycopg2.sql import Identifier
     from psycopg2.sql import SQL
 
-    query = SQL("""select 'one' as x""")
+    query = SQL("""select 'one' as {}""").format(Identifier("x"))
```

Comment:
This case fails on latest before these changes.

After these changes, it removes the double query.